experimenter/tests/integration/nimbus/android/update-fenix-apks.sh

```diff
@@ -12,3 +12,5 @@ TASK_ID=$(curl ${CURLFLAGS[@]} "${TASKCLUSTER_API}/tasks/${INDEX_BASE}" | jq '.t
 echo TASK ID "${TASK_ID}"
 
 echo "FIREFOX_FENIX_TASK_ID=${TASK_ID}" > fenix-build.env
+
```

Comment:
Could you move this whole script up into `./scripts/` at the top of the repo?

Reply:
Oh gee, totally forgot about that folder.